pkg/users/password-dialogs.js

```diff
@@ -136,7 +136,7 @@ function SetPasswordDialogBody({ state, errors, change }) {
     } = state;
 
     return (
-        <Form isHorizontal>
+        <Form isHorizontal onSubmit={apply_modal_dialog}>
```

Comment:
This is here just for the case where 'Old password' is missing I guess?

Reply:
I just added it to all <Form>s that I could find, regardless of whether they have a single input or not.